pkg/controllers/uiplugin/monitoring.go

```diff
@@ -88,28 +66,99 @@ func createMonitoringPluginInfo(plugin *uiv1alpha1.UIPlugin, namespace, name, im
 					Port:      9443,
 				},
 			},
-			{
-				Type:      "Service",
-				Alias:     "alertmanager-proxy",
-				Authorize: true,
-				Service: osv1alpha1.ConsolePluginProxyServiceConfig{
+		},
+	}
+
+	if persesDashboardsFeatureEnabled && !invalidPersesConfig {
```

Comment:
this logic seems a bit complex to maintain, we can maybe fail first if Perses or ACM configuration is not set

Reply:
In the latest commit, I updated the logic in monitoring.go. It validates and fails first if Perses or ACM configuration is not set.